bigbluebutton-html5/private/config/settings.yml

```diff
@@ -156,17 +157,6 @@ public:
       - setPresentation
       - zoomSlide
       - requestPresentationUploadToken
-      - transferUser
-      - createBreakoutRoom
-      - startPoll
```

Comment:
I think you removed a few lines too many when resolving the conflict.
I believe `startPoll, stopPoll, publishPoll` and `poll.max_custom` should remain in the file